### PR TITLE
Update categories.txt with danish translations

### PR DESCRIPTION
**Pull request by Lensi**

### CODE REVIEW

File: data/categories.txt [hunk 1]
```diff
@@ -7213,4 +7213,4 @@ sk:Štúdio jogy
 es:Estudio de yoga
 pl:Studio jogi
 nl:Yogastudio
-pt:Estúdio de ioga
\ No newline at end of file
+pt:Estúdio de ioga
```
Comment (Zverik) [inline data/categories.txt]: What has happened in this line?

**Comment by Zverik** [comment]
Could you please sign [the CLA](http://maps.me/ru/cla/) so we can use your commit?

File: data/categories.txt (hunk 1 above)
Comment (Lensi) [inline data/categories.txt]: I used Github's web editor. It looks like it added a newline to the end of the file.

**Comment by Lensi** [comment]
Signed, sealed, delivered.

File: data/categories.txt [hunk 2]
```diff
@@ -1131,7 +1131,7 @@ shop-mall
 en:Mall|5shopping mall|shop
 ar:ﻝﻮﻣ|المتجر
 cs:Obchoďák|obchod
-da:Centeret|butik
+da:Indkøbcenter|butikscenter|storcenter|center|butik
```
Comment (Zverik) [inline data/categories.txt]: Should it be `Indkøbscenter`?

Comment (Lensi) [inline data/categories.txt]: Yes, good catch! I corrected it.

**Comment by Zverik** [comment]
Thanks for following through, merging now :)

**Pull request merged**